django_eventstream/eventstream.py

```diff
@@ -58,6 +70,18 @@ def send_event(
         blocking=(not async_publish),
     )
 
+    # Publish event to Redis Pub/Sub if enabled
+    if redis_client:
```

Comment:
I suggest moving this section to be before `publish_event`, since `publish_event` can optionally block